src/olympia/accounts/tests/test_views.py

```diff
@@ -994,3 +1003,40 @@ def test_basic(self):
 
         with self.assertRaises(NotImplementedError):
             self.client.get(self.url)
+
+
+class TestSessionView(TestCase):
+    def login_user(self, user):
+        identity = {
+            'username': user.username,
+            'email': user.email,
+            'uid': user.fxa_id,
+        }
+        self.initialize_session({'fxa_state': 'myfxastate'})
+        with mock.patch(
+                'olympia.accounts.views.verify.fxa_identify',
+                lambda code, config: identity):
+            response = self.client.get(
+                '{url}?code={code}&state={state}'.format(
+                    url=reverse('accounts.authenticate'),
+                    state='myfxastate',
+                    code='thecode'))
+            cookie = response.cookies[views.JWT_TOKEN_COOKIE].value
+            assert cookie
+            verify = VerifyJSONWebTokenSerializer().validate({'token': cookie})
+            assert verify['user'] == user
+            assert self.client.session['_auth_user_id'] == str(user.id)
+            return cookie
+
+    def test_delete_when_authenticated(self):
+        user = user_factory(fxa_id='123123412')
+        token = self.login_user(user)
+        authorization = 'Bearer {token}'.format(token=token)
```

Comment:
Correct me if I'm wrong but both the docs and the code I found mention the ’JWT ...’ header prefix instead of ’Bearer’ which is used for something else. I think this needs changing and investigation why it's not failing?